pkg/eraser/eraser_test.go

```diff
@@ -64,7 +65,7 @@ func TestParseEndpointWithFallBackProtocol(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		p, a, e := parseEndpointWithFallbackProtocol(tc.endpoint, tc.fallbackProtocol)
+		p, a, e := util.ParseEndpointWithFallbackProtocol(tc.endpoint, tc.fallbackProtocol)
```

Comment:
Typically, tests will be in the same package as the package they're testing. The tests that are testing functions in `util` should be moved to `pkg/utils/utils_test.go` and have `package utils` (see my other comment about the package name).